Docs.md

```diff
@@ -339,6 +339,9 @@ Extra steps for Windows users:
     cd deno
     ./tools/setup.py
 
+    # For mac users, make sure that sccache is running by uncommenting the following line:
+    # prebuilt/mac/sccache --start-server
```

Comment:
Hmm - this isn't necessary for me. What happens if you don't do that?

Reply:
If I am not running `sccache` explicitly, it would fail with the message in https://github.com/denoland/deno/pull/1593#issuecomment-459046765 I took this from the Travis script: https://github.com/denoland/deno/blob/0be9d1ff0e2d07df82698661265c28a75603b5fe/.travis.yml#L63

Reply:
Ok - I've adjusted the comment in the patch to make it less decisive. I'll investigate this at some point. It ought to start the server itself.

Reply:
Sounds good to me! I did try to figure out how to make it automatically run when invoking `tools/build.py`, but sadly couldn't figure it out.